packages/flutter_tools/lib/src/template.dart

```diff
@@ -122,6 +124,8 @@ class Template {
 
       finalDestinationFile.writeAsBytesSync(sourceFile.readAsBytesSync());
     });
+
+    printStatus('$createdFileCount files created.');
```

Comment:
Not sure. When we only had a few files, it gave the user a sense of what the tool did, and which files were created. It worked great for 3-4 files; less so for 28.

I thought about listing individual top-level files and directories; that would be about ~10 items.